tools/crm_mon.c

```diff
@@ -4439,8 +4444,8 @@ clean_up(crm_exit_t exit_code)
             fprintf(stdout, "Content-Type: text/plain\n"
                             "Status: 500\n\n");
         } else {
-            crm_help('?', CRM_EX_USAGE);
+            return crm_help('?', CRM_EX_USAGE);
         }
     }
-    crm_exit(exit_code);
+    return crm_exit(exit_code);
 }
```

Comment:
`__attribute__((noreturn))` is a typical solution to this;
note that semantically the same is also standardized in C11,
so hopefully the static analyzers are keeping up.

Reply:
good point